.github/workflows/cicd.yml

```
@@ -107,6 +107,7 @@ jobs:
             org.opencontainers.image.licenses=MIT
             org.opencontainers.image.documentation=https://github.com/TryGhost/ActivityPub
             org.opencontainers.image.source=https://github.com/TryGhost/ActivityPub
+          platforms: linux/amd64,linux/arm64
```

actionlint (1.7.7)

input "platforms" is not defined in action "docker/metadata-action@v5". available inputs are "annotations", "bake-target", "context", "flavor", "github-token", "images", "labels", "sep-annotations", "sep-labels", "sep-tags", "tags"

(action)